guides/v2.2/cloud/project/project-multi-sites.md

```diff
@@ -8,157 +8,246 @@ functional_areas:
 
 #### To pass variables using the `magento-vars.php` file:
 
-1.  Open the `magento-vars.php` file in a text editor.
+1. Open the `magento-vars.php` file in a text editor.
+
+1. Uncomment everything after the first two lines.
 
-1.  Uncomment everything after the first two lines.
+1. Move the entire block starting with `if (isHttpHost("example.com")` after `function isHttpHost($host)`.
 
-1.  Move the entire block starting with `if (isHttpHost("example.com")` after `function isHttpHost($host)`.
+   The [default `magento-vars.php` file](https://github.com/magento/magento-cloud/blob/master/magento-vars.php) should look like the following:
 
-    The file should look like the following:
-    
-    ```php
-    // enable, adjust and copy this code for each store you run
-    // Store #0, default one
-    function isHttpHost($host)
-    {
-        if (!isset($_SERVER['HTTP_HOST'])) {
+   ```php
+   <?php
+   // enable, adjust and copy this code for each store you run
+   // Store #0, default one
+   //if (isHttpHost("example.com")) {
+   //    $_SERVER["MAGE_RUN_CODE"] = "default";
+   //    $_SERVER["MAGE_RUN_TYPE"] = "store";
+   //}
+   function isHttpHost($host)
+   {
+       if (!isset($_SERVER['HTTP_HOST'])) {
            return false;
-        }
-           return strpos(str_replace('---', '.', $_SERVER['HTTP_HOST']), $host) === 0;
-    }
-    if (isHttpHost("example.com")) {
-        $_SERVER["MAGE_RUN_CODE"] = "default";
-        $_SERVER["MAGE_RUN_TYPE"] = "store";
-    }
-    ```
-
-1.  Change the following line:
-
-    From:
-
-    ```php
-    return strpos(str_replace('---', '.', $_SERVER['HTTP_HOST']), $host) === 0;
-    ```
-
-    To:
-
-    ```php
-    return $_SERVER['HTTP_HOST'] ===  $host;
-    ```
-
-1.  Replace the following values in the `if (isHttpHost("example.com"))` block:
-
-    -  `example.com`—with the base URL of your website
-    -  `default`—with the unique code for your website or store view
-    -  `store`—with one of the following values:
-        -  `website`—load the website in the storefront
-        -  `store`—load a storeview in the storefront
-
-    ```php
-    // enable, adjust and copy this code for each store you run
-    // Store #0, default one
-    function isHttpHost($host)
-    {
-        if (!isset($_SERVER['HTTP_HOST'])) {
-        return false;
-        }
-        return $_SERVER['HTTP_HOST'] ===  $host;
-    }
-    if (isHttpHost("french.branch-sbg7pPa-f3dueAiM03tpy.us.magentosite.cloud"))
-    {
-        $_SERVER["MAGE_RUN_CODE"] = "french";
-        $_SERVER["MAGE_RUN_TYPE"] = "website";
-    }
-    ```
-
-1.  Save your changes to the `magento-vars.php` file.
+       }
+       return strpos(str_replace('---', '.', $_SERVER['HTTP_HOST']), $host) === 0;
+   }
+   ```
+   {:.no-copy}
+
+1. Change the following line:
+
+   From:
+
+   ```php?start_inline=1
+   return strpos(str_replace('---', '.', $_SERVER['HTTP_HOST']), $host) === 0;
```

Comment:
I guess we need to change it in the file itself. Do not see reasons to keep example with `---`